### DISCUSSION TITLE
CalDAV synchronization

### DISCUSSION THREAD

**Discussion by azmeuk** [GothenburgBitFactory/timewarrior#604] [Ideas]
A feature that I am missing is easy synchronization with a CalDAV server.
On a normal day of work I use 3 different computers, so synchronization is really appreciable.
I did not found a lot of discussion about this. There is the [caldavwarrior](https://gitlab.com/BlackEdder/caldavwarrior) project, but it synchronizes taskwarrior.
What do you think?

**Comment by lauft** [comment]
Synchronization with CalDAV requires implementation of a client to handle communication with the server and the conversion of Timewarrior data back and forth. This client could be integrated as a extension to Timewarrior. Currently, Timewarrior has no hooks which are executed automatically on certain events, so synchronization would be a manual step at first. 🤔 

There is https://github.com/timewarrior-synchronize for synchronization of Timewarrior data, but it is not done via CalDAV.
